app/models/dataset.rb

```diff
@@ -17,36 +18,16 @@ class Dataset < ApplicationRecord
 
   friendly_id :slug_candidates, :use => :slugged, :slug_column => :name
 
-  validates :frequency, inclusion: %w(daily weekly monthly quarterly annually financial-year never),
+  validates_inclusion_of :frequency, :in => %w(daily weekly monthly quarterly annually financial-year never),
                         allow_nil: true # To allow creation before setting this value
-
-  validates :title,
-    presence: { message: "Please enter a valid title" }
-
-  validates_format_of :title, {
-    with: /([a-z]){3}.*/i,
-    message: "Please enter a valid title"
-  }
-
-  validates :summary,
-    presence: { message: "Please provide a summary" }
-
-  validates :frequency,
-    presence: { message: "Please indicate how often this dataset is updated" },
-    if: lambda { published }
-
-  validates :licence,
-    presence: { message: "Please select a licence for your dataset" },
-    if: lambda{ published }
-
-  validates :licence_other,
-    presence: { message: "Please type the name of your licence" },
-    allow_blank: false,
-    if: lambda { licence == 'other' }
-
+  validates_presence_of :title
+  validates_format_of :title, :with => TITLE_FORMAT
```

Comment:
`:symbol => value` should always be written as `symbol: value` 

https://github.com/bbatsov/ruby-style-guide#symbols-as-keys